packages/components/ng-ovh-payment-method/src/components/integration/inContext/paypal/directive.js

```diff
@@ -39,7 +39,7 @@ export default {
     integrationCtrl.loading.init = true;
 
     // insert paypal checkout.js script if not yet loaded
-    if (!document.getElementById(PAYPAL_SCRIPT.id)) {
+    if (typeof paypal === 'undefined') {
```

Comment:
any reason for not using
```js
if (!paypal)
````
or
```js
if (paypal === undefined)
```

Reply:
No specific reason. All these 3 approaches works.

Reply:
done